src/api/explorer.js

```diff
@@ -22,46 +22,140 @@ const receiptsQuery = `
     }
 `;
 
-const getApi = () => {
-    return axios.create({
-        baseURL: process.env.REACT_APP_EXPLORER,
-    });
-}
+const getDollarsTransfersQuery = `
+  query Transfers($orderBy: [TransfersOrderBy!], $filter: TransferFilter) {
+    query {
+      transfers(orderBy: $orderBy, filter: $filter) {
+        nodes {
+          id
+          fromId
+          toId
+          value
+          extrinsicIndex
+          eventIndex
+          block {
+            id
+            number
+            timestamp
+          }
+        }
+      }
+    }
+  } 
+`;
 
-const palletToAsset = (pallet) => {
-    if (pallet == "ethLLMBridge") {
-        return "LLM";
-    } else if (pallet == "ethLLDBridge") {
-        return "LLD";
-    } else {
-        throw new Error("Unknown pallet");
+const getMeritsTransfersQuery = `
+  query Transfers($orderBy: [TransfersOrderBy!], $filter: TransferFilter) {
+    query {
+      transfers(orderBy: $orderBy, filter: $filter) {
```

Comment:
This fetches `LLD` transfers, not `LLM` transfers. LLM transfers are called `Merits` in GraphQL schema.